index.js

```diff
@@ -8,6 +8,7 @@ var path = require('path');
 var dirname = path.dirname;
 var basename = path.basename;
 var fs = require('fs');
+var readlink = require('readlink');
```

Comment:
Can you please inline the synchronous part of the `readlink` module; as module itself introduces an `async` module which is unnecessary for the commander and only introduce a ton of unused kb.

Reply:
I don't think that <kbd>CTRL+C</kbd>+<kbd>CTRL+V</kbd> code from `readlink` is good idea, because we must also provide tests and support this legacy code. If don't like `async` module maybe better send PR to `readlink` repo to remove it from dependencies.